src/windows/wsladiag/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+set(SOURCES main.cpp)
```

Comment:
FYI We'll probably want to add this new executable to the MSI package. 

To that, we'll need a add dependency in: https://github.com/microsoft/WSL/blob/feature/wsl-for-apps/msipackage/CMakeLists.txt (so that the MSI package depends on this new `wsladiag` target

And then add an entry in: [package.wix.in](https://github.com/microsoft/WSL/blob/feature/wsl-for-apps/msipackage/package.wix.in) (most likely something similar  to [this](https://github.com/microsoft/WSL/blob/8eee7439b2f1a7bbb0a6d5da04c1bce7b7ec8a0c/msipackage/package.wix.in#L29))